internal/resources/azure/app_service_plan.go

```diff
@@ -0,0 +1,114 @@
+package azure
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"fmt"
+	"strings"
+
+	"github.com/shopspring/decimal"
+)
+
+type AppServicePlan struct {
+	Address     string
+	SKUSize     string
+	SKUCapacity int64
+	Kind        string
+	Region      string
+}
+
+var AppServicePlanUsageSchema = []*schema.UsageItem{}
+
+func (r *AppServicePlan) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+func (r *AppServicePlan) BuildResource() *schema.Resource {
+	sku := r.SKUSize
+	skuRefactor := ""
```

Comment:
```suggestion
	var sku string
```

I don't think we need to assign `r.SKUSize` to anything, we can just reference it. We can then rename `skuRefactor` to `sku` since it's the actual SKU name.

Reply:
Fixed